File: app/src/main/java/com/intelegain/agora/model/TechnologyData.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by maninder on 21/11/17.
 */

public class TechnologyData {
    @SerializedName("TechnologyData")
    @Expose
    public List<TechnologyDatum> technologyData = null;


    public class TechnologyDatum {

        @SerializedName("techId")
        @Expose
        public Integer techId;
        @SerializedName("techName")
        @Expose
        public String techName;
        @SerializedName("subtechName")
        @Expose
        public String subtechName;

    }
}

File: app/src/main/java/com/intelegain/agora/model/TechnologyData.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by maninder on 21/11/17.
 */
class TechnologyData {
    @SerializedName("TechnologyData")
    @Expose
    var technologyData: List<TechnologyDatum>? = null

    inner class TechnologyDatum {
        @SerializedName("techId")
        @Expose
        var techId: Int? = null
        @SerializedName("techName")
        @Expose
        var techName: String? = null
        @SerializedName("subtechName")
        @Expose
        var subtechName: String? = null
    }
}